olp-cpp-sdk-core/include/olp/core/cache/CacheSettings.h

```diff
@@ -86,6 +86,16 @@ struct CacheSettings {
    * @brief Set the disk cache open options.
    */
   OpenOptions openOptions = OpenOptions::Default;
+
+  /**
+   * @brief Path to the protected (readonly) cache. This cache is used as a
+   * primary cache for lookup. DefaultCache will open this cache in read-only
+   * mode and will not write to it in case the disk_path_mutable is empty. This
+   * path should only be used in case users would like to have a stable fallback
+   * state or offline data that they can always access regardless of the network
+   * state.
+   */
+  boost::optional<std::string> disk_path_protected = boost::none;
```

Comment:
The option is not used anywhere, is it?

Reply:
Not, yet! But things will change.

Reply:
Why not wait until the usage of the cash is introduced as well?

Reply:
Yeah, the ticket is about to introduce a new parameter only. The usage will be in another reviews/tickets.